uk.co.compucorp.civicrm.hrcore/js/src/hrcore/helpers/help-icon-injector.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env amd */
+
+(function (CRM) {
+  define(function () {
+    'use strict';
+
+    return injectHelpIcon;
+
+    /**
+     * Injects a help icon to a form field
+     *
+     * @param {jQuery} formClass - a class of the CiviCRM form
```

Comment:
```js
* @param {jQuery} formClass a class of the CiviCRM form
```
i don't believe we're using the `-` as a separator (both formats are valid jsdoc formats as far as i know)

Reply:
Done